src/test/java/codeu/model/data/ActivityTest.java

```diff
@@ -0,0 +1,53 @@
+package codeu.model.data;
+
+import codeu.model.util.Util;
+import org.junit.Test;
+
+import java.time.Instant;
+import java.util.UUID;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class ActivityTest {
+  @Test
+  public void testCreate1() {
+    UUID id = UUID.randomUUID();
+    UUID ownerId =id;
+    Action action = Action.REGISTER_USER;
+    Boolean isPublic = true;
+    Instant creation = Instant.now();
```

Comment:
at Line 16, is that `.REGISTER_USER` a part of the enum class you implemented for the activities?

Reply:
Sorry I did not see your comments:

Yes, it is one of the actions a user can take!